packages/api/src/command/medical/admin/upload-doc.ts

```diff
@@ -6,7 +6,8 @@ import { getPatientOrFail } from "../patient/get-patient";
 import { Config } from "../../../shared/config";
 import { createMetriportDocReferenceContent } from "../../../external/fhir/document";
 
-const docContributionUrl = Config.getDocContributionUrl();
+const apiUrl = Config.getApiUrl();
+const docContributionUrl = `${apiUrl}/doc-contribution/commonwell/`;
 
```

Comment:
Thought about adding this to the env config as its own params but since its using the apiUrl thought this was simpler. Let me know what you guys think